pom.xml

```diff
@@ -48,9 +48,9 @@
     <version.system-rules>1.16.1</version.system-rules>
     <version.javassist>3.21.0-GA</version.javassist>
     <version.jgrapht>1.0.1</version.jgrapht>
-    <version.shrinkwrap.resolver>3.0.0-alpha-4</version.shrinkwrap.resolver>
-    <version.snakeyaml>1.18</version.snakeyaml>
-    <version.mockito>2.10.0</version.mockito>
+    <version.shrinkwrap.resolver>3.0.0-beta-2</version.shrinkwrap.resolver>
```

Comment:
it is strange to release something with a dependency to a beta, but well, before it was an alpha :)

Reply:
it's almost done and totally stable... When I release the final version I'll update it ;-)